Gifski/InternetAccessPolicy.json

```diff
@@ -8,7 +8,7 @@
 			"NetworkProtocol" : "TCP",
 			"Port" : "443",
 			"Purpose" : "Gifski sends crash reports to this server.",
-			"DenyConsequences" : "If you deny this connection, the developers of Gifski won't be notified of any crashes and they can't fix what they don't know about."
```

Comment:
I felt like the "and they can't fix what they don't know about" is a little too condescending, even though we don't have that intentions in mind. Not sure on the fix but I think we should update the sentence to be a little more nicer than this :D

Reply:
I agree. This sounds nicer.